src/lib/ecore_evas/ecore_evas_private.h

```diff
@@ -442,30 +418,30 @@ struct _Ecore_Evas_Aux_Hint
    unsigned char notified : 1; // let caller know ee has got response for this aux hint
 };
 
-EAPI void _ecore_evas_ref(Ecore_Evas *ee);
-EAPI void _ecore_evas_unref(Ecore_Evas *ee);
-EAPI int ecore_evas_buffer_render(Ecore_Evas *ee);
-
-EAPI void _ecore_evas_fps_debug_init(void);
-EAPI void _ecore_evas_fps_debug_shutdown(void);
-EAPI void _ecore_evas_fps_debug_rendertime_add(double t);
-EAPI void _ecore_evas_register(Ecore_Evas *ee);
-EAPI void _ecore_evas_subregister(Ecore_Evas *ee_target, Ecore_Evas *ee);
-EAPI void _ecore_evas_register_animators(Ecore_Evas *ee);
-EAPI void _ecore_evas_free(Ecore_Evas *ee);
-EAPI void _ecore_evas_idle_timeout_update(Ecore_Evas *ee);
-EAPI void _ecore_evas_mouse_move_process(Ecore_Evas *ee, int x, int y, unsigned int timestamp);
-EAPI void _ecore_evas_mouse_device_move_process(Ecore_Evas *ee, Efl_Input_Device *pointer,
+ECORE_EVAS_API void _ecore_evas_ref(Ecore_Evas *ee);
+ECORE_EVAS_API void _ecore_evas_unref(Ecore_Evas *ee);
+ECORE_EVAS_API int ecore_evas_buffer_render(Ecore_Evas *ee);
+
+ECORE_EVAS_API void _ecore_evas_fps_debug_init(void);
```

Comment:
before someone asks, I checked this some days ago and I found that all this `_functions` seems to be intern but actually needs to be exported.